project/CLI.py

```diff
@@ -107,12 +107,19 @@ def create(self, args):
 
 def perform_team_edit(self, args):
     try:
-        if args[1] == "name":
-            if self.game.modify_team(args[2], args[4]):
+        if args[2] == "name" and len(args) < 5:
```

Comment:
I feel like there's a more elegant way to handle this, like maybe find the index of 'name' and use +1 to get the value... args is a list, so we could also use 'name' in args somehow.

Reply:
I am not sure what you mean here. But the other suggestions have been added.